contracts/chains/arbitrum/router/IporProtocolRouterArbitrum.sol

```diff
@@ -81,20 +85,20 @@ contract IporProtocolRouterArbitrum is IporProtocolRouterAbstract {
     function getConfiguration() external view returns (DeployedContractsArbitrum memory) {
         return
             DeployedContractsArbitrum({
-                ammSwapsLens: ammSwapsLens,
-                ammOpenSwapServiceWstEth: ammOpenSwapServiceWstEth,
-                ammCloseSwapLens: ammCloseSwapLens,
-                ammCloseSwapServiceWstEth: ammCloseSwapServiceWstEth,
-                ammGovernanceService: ammGovernanceService,
-                liquidityMiningLens: liquidityMiningLens,
-                powerTokenLens: powerTokenLens,
-                flowService: flowService,
-                stakeService: stakeService,
-                ammPoolsServiceWstEth: ammPoolsServiceWstEth,
-                ammPoolsLensWstEth: ammPoolsLensWstEth,
-                ammPoolsServiceUsdm: ammPoolsServiceUsdm,
-                ammPoolsLensUsdm: ammPoolsLensUsdm
-            });
+            ammSwapsLens: ammSwapsLens,
```

Comment:
ask frontend if they use this ?

Reply:
we are using `ammOpenSwapServiceWstEth` adddress field to read `openingFeeRate` and bunch of other params, how can we read it now? Should we take the address that is stored in "wstEth: wstEth" param?